### PR TITLE
Adding book.toml for mdbook

### PR DESCRIPTION
**Pull request by fel115**
Issue #24.

I added the book.toml file for mdbook. Should I also push the generated files from mdbook ?

### CODE REVIEW

**Review by markus2330 — CHANGES_REQUESTED** [review]
Doesn't work for me, I get:

```
2023-07-17 15:35:16 [ERROR] (mdbook::utils): Error: Couldn't open SUMMARY.md in "/home/markus/Projekte/Opensesame/opensesame/doc" directory
2023-07-17 15:35:16 [ERROR] (mdbook::utils):    Caused By: No such file or directory (os error 2)
```

SUMMARY.md needs to list all the files we want to include.

The generated book (html) should not be checked in, eventually the CI will build and publish it.

**Comment by fel115** [comment]
It should work now. I forgot to add the SUMMARY.md file in the PR.

File: doc/SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Summary
```
Comment (markus2330) [inline doc/SUMMARY.md:1]: Here should be a list of files in the order they should be shown. Please go over the documents and check if the:

- title is okay
- content is formatted properly

e.g. doc/Install.md should be the first document, with `Install` (or similar) as title and nice code formatting throughout.

**Review by markus2330 — CHANGES_REQUESTED** [review]

File: doc/SUMMARY.md (hunk 1 above)
Comment (fel115) [reply doc/SUMMARY.md:1]: Is this order okay?

File: doc/SUMMARY.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- [I2C](I2C.md)
+- [MQ135](MQ135.md)
+- [Garage](Garage.md)
+- [Environment](Environment.md)
+- [Front door](Haustüre.md)
+- [Weather station](thiesclima-weatherstation.md)
+- [Inverter](goodwe-inverter.md)
+- [Troubles libmodbus](libmodbus-troubles.md)
+- [Energy](Energy.md)
+- [Translations](Translastions.md)
```
Comment (markus2330) [inline doc/SUMMARY.md:17]:
```suggestion
- [Translations](Translations.md)
```

**Review by markus2330 — APPROVED** [review]
Thank you, much better!

**Pull request merged**